playing-card-detector/newProject/serverClient.py

```diff
@@ -34,6 +36,22 @@ def main():
         data = connection.recv(518400)
         if data:
             print("get data")
+
+            img = np.frombuffer(data, dtype='uint8')
+            img = np.reshape(img, (540, 960))
+            #img = Image.fromarray(img)
```

Comment:
If the comments are supposed to be here, maybe add a comment explaining what the commented out code is meant for. Else, it could probably be removed.

What if the data received has other dimensions than 540x960?

Reply:
that is a quick change that is still a work in progress and will be changed shortly, I really can't be bothered to fix this atm